code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -1027,6 +1027,7 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
 	listed_products = list(
 		/obj/effect/essentials_set/engi = list(CAT_ESS, "Essential Engineer Set", 0, "white"),
 
+		/obj/item/storage/backpack/lightpack = list(CAT_ENGSUP, "Combat Backpack", 15, "orange"),
```

Comment:
```suggestion
		/obj/item/storage/backpack/lightpack = list(CAT_ENGSUP, "Combat Backpack", 26, "orange"),
```
best bag in the game for 15 points good meme, able to also get cutter
26 for cutter or this

Reply:
yeah but if you get a cutter and this, you get barely any cells, cells aren't orderable either

you also lose your access to fuel (since if you take combat backpack, you don't have welder pack)

spending 26 points on a combat backpack is really not worth it; it just becomes a noob trap since thats half your points that could have been spent on materials instead, for instance